docs/source/index.rst

```diff
@@ -11,6 +11,14 @@ AppTools Documentation
     selection/*
     naming/*
     io/*
-    api
+
+API Documentation
+-----------------
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
```

Comment:
Is this `:glob:` needed?
(I believe the `:glob:` in the above section is for the use of `*`. But here `*` is not used.)